docs/tech-reference/yum-plugins.rst

```diff
@@ -622,6 +622,8 @@ configuration values are optional.
 ``recursive`` Boolean flag. If true, units are copied together with their
 dependencies. False by default.
 
+``recursive_conservative`` Boolean flag. If true, units are copied together with their
+dependencies, target repository content is considered. False by default.
```

Comment:
Perhaps you should modify the description of recursive also. We want to state that it pulls in the latest version of dependencies. Then here you can say this does not copy any deps already satisfied by target repository.